packages/composite-checkout/src/components/checkout-steps.tsx

```diff
@@ -56,7 +56,7 @@ const noop = () => {
 const noopPromise = () => () =>
 	Promise.reject( 'Cannot use CheckoutStepGroupContext without a provider.' );
 const emptyStepCompleteStatus = {};
-const CheckoutStepGroupContext = createContext< CheckoutStepGroupStore >( {
+export const CheckoutStepGroupContext = createContext< CheckoutStepGroupStore >( {
```

Comment:
@sirbrillig Thoughts on allowing this to be exported? I did this because I needed access to the setActiveStepNumber action in order to have the "Remove filter" button open the Billing Information step, but I am not sure if this is the only way to do that?

My reasoning for sending the customer back to the Billing Information step is so that they have to explicitly deselect the checkbox to stop filtering by business cards.

My worry here is that CheckoutStepGroupContext is _not_ exported because we wish to keep it scoped to the checkout-steps file 🤔

Reply:
Yes, your instinct is right. Exporting this context opens a can of worms in terms of the internal business of the checkout step system.

Instead, we can just provide a new hook to access the step setting behavior. I've added one in https://github.com/Automattic/wp-calypso/pull/101753